src/input_source.h

```diff
@@ -68,13 +73,22 @@ namespace Input {
 
 		bool InitRecording(const std::string& record_to_path);
 
+		void GetMousePosition(int& x, int& y) const {
```

Comment:
We have  `Point` struct now which contains x and y, if you don't want to use out params here.

Reply:
Valid "Point". I changed this mouse_x/y to Point everywhere.